### DISCUSSION TITLE
Moved Site to Wordpress - Data is Pulling from old site

### DISCUSSION THREAD

**Discussion by Goonysings** [code4recovery/12-step-meeting-list#624] [Q&A] [answered]
I have published our site on Wordpress and have been checking the App. The only difference in data is the addition of our Online groups, since they are missing from the app, I assume that the data is pulling from the previous site/host location. How do I make this change? 

Go slow... I'm limited in knowledge. 

Our site: www.district70aa.org

**Comment by tech2serve** [comment, marked as answer]
Hi,

You will probably need to contact Meeting Guide app support to get the change published. Please check out their new [support site](https://meetingguide.helpdocs.io) and use the contact form to reach them.

Tim/Tech2Serve

**Comment by Goonysings** [comment]
Thank you! I'm all set.

On Mon, Jan 3, 2022 at 9:23 AM Tech2Serve ***@***.***> wrote:

> Hi,
>
> You will probably need to contact Meeting Guide app support to get the
> change published. Please check out their new support site
> <https://meetingguide.helpdocs.io> and use the contact form to reach them.
>
> Tim/Tech2Serve
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/discussions/624#discussioncomment-1900654>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AWDZHNFL2YOZUCAHPYNBTX3UUG5QTANCNFSM5LFNRCPA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you authored the thread.Message ID:
> <code4recovery/12-step-meeting-list/repo-discussions/624/comments/1900654@
> github.com>
>


-- 

-- 

Leann Nelson

***@***.*** | 815-980-6865 | byleann.com

[image: Twitter]  <https://twitter.com/leannrockford> [image: Facebook]
<https://facebook.com/leannrockford> [image: LinkedIn]
<https://www.linkedin.com/in/leannnelson>

www.wwwwwww
